core/local/steps/add_infos.js

```diff
@@ -33,11 +37,11 @@ module.exports = function (buffer /*: Buffer */, opts /*: { syncPath: string } *
             event.kind = event.kind === 'directory' ? 'directory' : 'file'
           }
         }
-        batch.push(event)
       } catch (err) {
         log.info({err, event}, 'Cannot get infos')
-        console.log('stats', err) // TODO error handling
+        event.incomplete = true
```

Comment:
Could the `try` / `catch` block wrap only the `stater.stat()` call?

Reply:
We could, but I prefer to not do that. I don't think it's a good idea to set the `kind` for an event when `stats` has failed.